pkg/cmd/server/kubernetes/master.go

```diff
@@ -133,3 +137,24 @@ func (c *MasterConfig) RunMinionController() {
 
 	glog.Infof("Started Kubernetes Minion Controller")
 }
+
+func (c *MasterConfig) createSchedulerConfig(configFactory *factory.ConfigFactory) (*scheduler.Config, error) {
```

Comment:
Does this ever get called from somewhere else?  If not, the `MasterConfig` contains sufficient information to eliminate the argument.